### ISSUE TITLE
Notifications: Error notification links are not contrast accessible

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/design-system#1391]
**Describe the bug**

Add links to the HTML on the error notifications and see that the link color is not accessible on the red background:

<img width="385" alt="Screen Shot 2022-08-26 at 12 27 02 PM" src="https://user-images.githubusercontent.com/704760/186950466-4b14b010-a0bd-415b-b6e3-0fe68bf1c8e4.png">

**Comment by anselmbradford** [comment]
Fixed by https://github.com/cfpb/design-system/pull/1944

**Comment by anselmbradford** [comment]
And https://github.com/cfpb/design-system/pull/1945
